pkgs/development/libraries/inchi/default.nix

```diff
@@ -0,0 +1,48 @@
+{ pkgs, fetchurl, stdenv, unzip }:
+stdenv.mkDerivation {
+  pname = "inchi";
+  version = "1.05";
+  src = fetchurl {
+    url = "http://www.inchi-trust.org/download/105/INCHI-1-SRC.zip";
+    sha1 = "e3872a46d58cb321a98f4fd4b93a989fb6920b9c";
+  };
+
+  nativeBuildInputs = [ pkgs.unzip ];
+  outputs = [ "out" "doc" ];
+
+  enableParallelBuilding = true;
+
+  buildPhase = ''
```

Comment:
If you just put `cd ./INCHI_API/libinchi/gcc` in `preBuild` then you don't need to overwrite the `buildPhase`.